### DISCUSSION TITLE
How is the "Node distances" plot drawn?

### DISCUSSION THREAD

**Discussion by D19CrMei** [talmolab/sleap#844] [Help!]
How was this picture drawn? I am getting blurry when the GUI interface is open, I want to know how such a clear result is drawn

![1658030951199](https://user-images.githubusercontent.com/73282211/179383615-864958f2-4795-45d1-b07a-c2969e60a366.jpg)

**Comment by roomrys** [comment]
Hi @D19CrMei,

That picture is drawn using [matplotlib](https://matplotlib.org/) by the [`_plot_distances`](https://github.com/talmolab/sleap/blob/254759d50d2ef1a0c2483b04233510718ec9f7e3/sleap/gui/dialogs/metrics.py#L316) function in SLEAP.

Thanks,
Liezl

**Reply by D19CrMei** [reply]
Yes, I also found it, but my result graph is blurry in the GUI, why is this? How can I modify it?

**Reply by D19CrMei** [reply]
I tried many times, but still can't modify the font size of the x,y coordinate scale, only the title size, how can I modify the code to get the same clear result as the above picture?
